source/manifest.json

```diff
@@ -35,7 +35,8 @@
 	"background": {
 		"scripts": [
 			"assets/background.js"
-		]
+		],
+		"persistent": false
```

Comment:
This has always been a requirement on iOS but we dropped it because it caused issues with GHE if I remember correctly. It has been added via the Safari preparation script, which needs to be updated as part of this PR as well, instead of altering the manifest.

Reply:
Ah ok. Reverted: https://github.com/refined-github/refined-github/pull/6874/commits/ab8efd73cde824b5135a2e9cf2df5b97d36a8685

I had not run the prepare script as I was not doing a new version.